aspnetcore/blazor/components/event-handling.md

```diff
@@ -142,6 +144,45 @@ For more information, see the following resources:
 
 * <xref:Microsoft.AspNetCore.Components.Web.EventHandlers> holds attributes to configure the mappings between event names and event argument types.
 
+:::moniker range=">= aspnetcore-8.0"
```

Comment:
We don't typically call out support for individual events on elements like this. We could try to exhaustively document all of the events that Blazor has built-in support for, but that seems a bit excessive. Should we just move this to the release notes doc?